app/assets/javascripts/application.js

```diff
@@ -3,7 +3,7 @@
 //= require jquery_ujs
 //= require jquery-ui
 //= require bootstrap-sprockets
-//= require bootstrap
+//////// require bootstrap
```

Comment:
Does the extra /'s make this into a commit, or does it still include bootstrap? What is the desired behavior?